alembic/versions/3d9c7f4c3e4c_add_dm_sent_field_to_infractions_model.py

```diff
@@ -6,21 +6,21 @@
 Create Date: 2021-11-10 22:26:00.101691
```

Comment:
Could you add all of these changes to the mako files, so that all future migrations get them when generated?